tests/interactive.py

```diff
@@ -6,7 +6,8 @@
 
 from __future__ import absolute_import
 import looper
-from . import conftest
+#from . import conftest
+import conftest
```

Comment:
Yeah I think that was the genesis but with the directory location mentioned in the comments it seems to be working with the bare/non-relative `conftest` import on both 2 and 3